modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/modelvalidation/v0.1.0.json

```diff
@@ -0,0 +1,54 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/modelvalidation/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:ModelValidationShape",
+      "@type": "sh:NodeShape",
+      "label": "Model validation shape",
+      "targetClass": "nsg:ModelValidation",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:used",
+              "name": "Model",
+              "description": "Used model.",
+              "seeAlso": "{{base}}/schemas/neurosciencegraph/simulation/modelinstance/v0.1.0/shapes/ModelInstanceShape",
+              "class": "nsg:ModelInstance",
+              "minCount": 1
+            },
+            {
+              "path": "prov:used",
+              "name": "Model",
```

Comment:
There seems to be some inadvertent duplication here

Reply:
The values of the "name" annotation are misleading here (I will fix them). The intent here is to say that a model validation required the model to be validated as well as a variable report. I'll be honest here: I don't know (yet) what a variable report is really but we've added it during the hackathon.